ee/vellum_ee/workflows/tests/test_server.py

```diff
@@ -120,11 +122,10 @@ class CodeExecutionNode(BaseCodeExecutionNode[BaseState, int]):
 
     namespace = str(uuid4())
 
-    # AND the virtual file loader is registered
-    sys.meta_path.append(VirtualFileFinder(files, namespace))
+    finder = VirtualFileFinder(files, namespace)
```

Comment:
[nit] let's keep this line under the ` # AND the virtual file loader is registered`